scripts/plot_run_duration_over_time.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Takes a single benchmark output JSON and plots the duration of item executions over time
+
+# TODO add to requirements.txt
+import json
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import sys
+
+if(len(sys.argv) != 2):
+    print('Usage: ' + sys.argv[0] + ' benchmark.json')
+    exit()
+
+with open(sys.argv[1]) as json_file:
+    data_json = json.load(json_file)['benchmarks']
+
+# Build flat list of {name, begin, duration} entries for every benchmark item run
```

Comment:
As is evident, I am not the biggest Python hacker. I can make things work, but there are probably various things that could be improved here. Please let me know!